packages/components/psammead-topic-tags/src/index.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { string, oneOf, shape, node } from 'prop-types';
+import styled from '@emotion/styled';
+import { C_LUNAR, C_EBON, C_METAL } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING_HLF, GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { getBrevier } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+
+const [TAG_TEXT_PAD_Y, TAG_TEXT_PAD_X, ROW_SPACING] = [
+  '0.375rem',
+  '0.4375rem',
+  '0.0625rem',
+];
+
+const topicTagsContainer = `
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: -${ROW_SPACING};
+  margin-bottom: 0;
+  margin-left: -${GEL_SPACING_HLF};
+  margin-right: -${GEL_SPACING_HLF};
+  padding: 0;
+`;
+
+const SingleTopicTagContainer = styled.div`
+  ${topicTagsContainer}
+`;
+
+const TopicsList = styled.ul`
+  list-style-type: none;
+  ${topicTagsContainer}
+`;
+
+const SingleTopicTagItem = styled.div`
+  ${({ service }) => getSansRegular(service)}
+  ${({ script }) => script && getBrevier(script)}
+
+  word-break: break-word;
+  margin-top: ${ROW_SPACING};
+  margin-left: ${GEL_SPACING_HLF};
+  margin-right: ${GEL_SPACING_HLF};
+  a {
+    display: flex;
+    padding-top: ${GEL_SPACING};
+    padding-bottom: ${GEL_SPACING};
+    text-decoration: none;
+    color: ${C_EBON};
+
+    &:hover,
+    &:focus {
+      text-decoration: underline;
+    }
+    &:visited {
+      color: ${C_METAL};
+    }
+  }
+
+  a > span {
+    background-color: ${C_LUNAR};
+    padding: ${TAG_TEXT_PAD_Y} ${TAG_TEXT_PAD_X};
+  }
+`;
+
+export const TopicTag = ({ topicName, topicLink }) => (
+  <a href={topicLink}>
+    <span>{topicName}</span>
+  </a>
+);
+
+export const TopicTags = ({ dir, children, script, service }) => {
+  if (children.length === 0) return null;
```

Comment:
These kinds of checks make a component less predictable. If I want to render this container, but haven't given it a child yet: do I go into debug mode, because it's simply not appearing on the page? Remember, I could have installed your package from NPM and have no idea how it's written. 

'[Pure functions](https://en.wikipedia.org/wiki/Pure_function)' return exactly one possible value for any given input. That means if I pass it zero or 10 children, a pure function would behave exactly the same way. Where possible, we should try to write functions that are pure - that includes functional React components. They are easier to reason and, for React, easier to optimise.